recipe/activate.sh

```diff
@@ -17,6 +17,26 @@ export CC_@CONDA_RUST_HOST_LOWER@="${CC_FOR_BUILD:-${CONDA_PREFIX}/bin/@rust_def
 
 if [[ "@cross_target_platform@" == linux*  ]]; then
   export CARGO_BUILD_RUSTFLAGS="-C link-arg=-Wl,-rpath-link,${PREFIX:-${CONDA_PREFIX}}/lib -C link-arg=-Wl,-rpath,${PREFIX:-${CONDA_PREFIX}}/lib"
+elif [[ "@cross_target_platform@" == win* ]]; then
+  export CARGO_TARGET_@rust_arch_env@_LINKER=${CONDA_PREFIX}/bin/lld-link
+
+  # some rust crates need a linux gnu c compiler at buildtime
+  # thus we need to create custom cflags since the default ones are for clang
+  export AR_@CONDA_RUST_HOST_LOWER@="${AR}"
+  export AR_@CONDA_RUST_TARGET_LOWER@=$CONDA_PREFIX/bin/llvm-lib
+
+  export CFLAGS_@CONDA_RUST_HOST_LOWER@=""
+  export CFLAGS_@CONDA_RUST_TARGET_LOWER@="${CFLAGS}"
+
+  export CPPFLAGS_@CONDA_RUST_TARGET_LOWER@="${CPPFLAGS}"
+  export CPPFLAGS_@CONDA_RUST_HOST_LOWER@=""
+
+  export CC_@CONDA_RUST_TARGET_LOWER@=$CONDA_PREFIX/bin/clang-cl
+  export CXX_@CONDA_RUST_TARGET_LOWER@=$CONDA_PREFIX/bin/clang-cl
+
+  # Setup CMake Toolchain
+  export CMAKE_GENERATOR=Ninja
+  export CMAKE_TOOLCHAIN_FILE_@CONDA_RUST_TARGET_LOWER@=$RECIPE_DIR/win_toolchain.cmake
```

Comment:
RECIPE_DIR?

Reply:
This is a leftover from when I set everything up in the Polars build. I will install the toolchain file into the prefix and update the activation script

Reply:
I don't understand why we need this just for windows.

Reply:
if we don't pass `/manifest:no` as a linker flag the build fails with:

```
      [2/2] : && /root/micromamba/envs/build-env/bin/cmake -E vs_link_exe --intdir=CMakeFiles/cmTC_6ee41.dir --rc=/root/micromamba/envs/build-env/bin/llvm-rc --mt=/usr/bin/mt --manifests  -- /root/micromamba/envs/build-env/bin/lld-link /nologo CMakeFiles/cmTC_6ee41.dir/testCCompiler.c.obj  /out:cmTC_6ee41.exe /implib:cmTC_6ee41.lib /pdb:cmTC_6ee41.pdb /version:0.0 /machine:x64 --target=x86_64-pc-windows-msvc19.37.32822 -nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -Wl,-defaultlib:/root/micromamba/envs/build-env/lib/clang/17/lib/windows/clang_rt.builtins-x86_64.lib  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && :
      FAILED: cmTC_6ee41.exe 
      : && /root/micromamba/envs/build-env/bin/cmake -E vs_link_exe --intdir=CMakeFiles/cmTC_6ee41.dir --rc=/root/micromamba/envs/build-env/bin/llvm-rc --mt=/usr/bin/mt --manifests  -- /root/micromamba/envs/build-env/bin/lld-link /nologo CMakeFiles/cmTC_6ee41.dir/testCCompiler.c.obj  /out:cmTC_6ee41.exe /implib:cmTC_6ee41.lib /pdb:cmTC_6ee41.pdb /version:0.0 /machine:x64 --target=x86_64-pc-windows-msvc19.37.32822 -nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -Wl,-defaultlib:/root/micromamba/envs/build-env/lib/clang/17/lib/windows/clang_rt.builtins-x86_64.lib  /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && :
      MT: command "/usr/bin/mt /nologo /manifest CMakeFiles/cmTC_6ee41.dir/intermediate.manifest /out:CMakeFiles/cmTC_6ee41.dir/embed.manifest /notify_update" failed (exit code 0x2) with the following output:
      /usr/bin/mt: invalid argument ‘/nologo’ for ‘operation’
      Valid arguments are:
        - ‘eof’, ‘weof’
        - ‘fsf’
        - ‘bsf’
        - ‘fsr’
        - ‘bsr’
        - ‘rewind’
        - ‘offline’, ‘rewoffl’, ‘eject’
        - ‘status’
        - ‘bsfm’
        - ‘eom’
        - ‘retension’
        - ‘erase’
        - ‘asf’
        - ‘fsfm’
        - ‘seek’
      ninja: build stopped: subcommand failed.
```

cargo-xwin does the same here: https://github.com/rust-cross/cargo-xwin/blob/main/src/common.rs#L445

Reply:
Following doesn't work?
```
export CARGO_BUILD_RUSTFLAGS="-C link-arg=/manifest:no"
```

Reply:
fails with the same error

Reply:
export LDFLAGS="$LDFLAGS -manifest:no"

Reply:
This does work!

Reply:
Let's move that to https://github.com/conda-forge/clang-win-activation-feedstock

Reply:
Also the `CMAKE_GENERATOR` ?

Reply:
The manifest LDFLAG is only valid for clang-cl, while CC and CXX is set to clang by the clang-win-activation-feedstock. I think we should leave it in the rust-activation-feedstock